skxray/tests/test_cdi.py

```diff
@@ -71,3 +73,106 @@ def test_convolution():
         g2 = gauss(v, std2)
         f = convolution(g1, g2)
         assert_almost_equal(0, np.mean(f), decimal=3)
+
+
+def test_fft_helper():
+    x = np.linspace(-2, 2, 100, endpoint=True)
+    g = np.exp(x ** 2 / 2)
+
+    g_fft = _fft_helper(g)
+    g_ifft = _fft_helper(g_fft, option='ifft')
+    assert_array_almost_equal(np.abs(g_ifft), g, decimal=10)
+
+
+def test_relative_error():
+    shape_v = [3, 3]
+    a1 = np.zeros(shape_v)
+    a2 = np.ones(shape_v)
+
+    e1 = cal_relative_error(a2, a1)
+    assert_equal(e1, 1)
+
+    e2 = cal_relative_error(a2, a2)
+    assert_equal(e2, 0)
+
+
+def test_find_support():
+    shape_v = [100, 100]
+    cenv = shape_v[0]/2
+    r = 20
+    a = np.zeros(shape_v)
+    a[cenv-r:cenv+r, cenv-r:cenv+r] = 1.0
+    sw_sigma = 0.5
+    sw_threshold = 0.01
+
+    new_sup, s_index, s_out_index = find_support(a, sw_sigma, sw_threshold)
+    # the area of new support becomes larger
+    assert(np.sum(new_sup)-np.sum(a) > 0)
+
+
+def test_pi_support():
+    a1 = np.ones([2, 2])
+    a1[0, 0] = 1
+    index = np.where(a1 == 1)
+    a2 = pi_support(a1, index)
+    assert_equal(np.sum(a2), 0)
+
+
+def cal_diff():
```

Comment:
Can you call this something like `make_synthetic_data` to make it clear what is going on?  This does something very different than `cal_diff_error`